doc/plugin_server_nodeattestor_awsiid.md

```diff
@@ -0,0 +1,13 @@
+# Server plugin: NodeAttestor "aws_iid"
+
+*Must be used in conjunction with the agent-side aws_iid plugin*
+
+The `aws_iid` plugin automatically attests instances using the AWS Instance Metadata API and the AWS Instance Identity document. It also allows an operator to use AWS Instance IDs when defining SPIFFE ID attestation policies. 
```

Comment:
Perhaps we should provide an example SPIFFE ID for agents using this attestor? For instance:

> Agents attested by the `aws_iid` attestor will be issued a SPIFFE ID like `spiffe://example.org/agent/aws_iid/ACCOUNT_ID/INSTANCE_ID`

Reply:
done